aws-athena/ctas/ccao-python_model_dependency.sql

```diff
@@ -0,0 +1,3 @@
+{{ config(materialized='table') }}
```

Comment:
This has to be a table because Athena views cannot be accessed in PySpark scripts.